projects/libs/core-library-typescript/src/public-api/transaction-handler.ts

```diff
@@ -0,0 +1,30 @@
+import { types } from "../common-library/types";
+
+export interface TransactionHandlerConfig {
+    validateSubscription(): boolean;
```

Comment:
Can you change the indentation to 2 spaces, like in (most of the) other files?

Reply:
don't we have an editor config file? this should be automatic

Reply:
We don't have one. You are right, it's should be added and automatic.